packages/teleterm/src/preload.ts

```diff
@@ -80,8 +78,7 @@ async function getElectronGlobals(): Promise<ElectronGlobals> {
   // All uses of tshClient must wait before waitForTshdEventsClient finishes. Otherwise we run into
   // a risk of causing panics in tshd due to a missing tshd events client.
```

Comment:
I forgot to update this comment after simplifying the setup. I'll do so when applying the feedback from code review.